docs/guides/exporting.md

```diff
@@ -212,6 +212,89 @@ marimo islands are a way to embed marimo outputs and/or python code in your HTML
 
 Check out an [example island-powered document](./island_example.md).
 
+You can use `MarimoIslandGenerator` to embed marimo "islands", or blocks of marimo code and outputs, in a static web page. For example, use this class in conjunction with other static site generators (SSGs) to embed marimo into their generated content.
+
+### Generating islands
```

Comment:
There are now two redundant sections called generating islands:

<img width="208" alt="image" src="https://github.com/user-attachments/assets/9bddcaac-43e0-472c-83ac-6ab3c70af252" />

Reply:
The documentation you've added is nice but you've already added it to the API docs, so this is redundant. I suggest deleting the entire `### generating islands` section you've added and just let people discover the API docs.